ci/job.yml

```diff
@@ -0,0 +1,168 @@
+- job:
+    name: 'centos-container-pipeline-service-container-index'
+    description: |
+        Managed by Jenkins Job Builder, do not edit manually!
+    node: container
+    properties:
+        - github:
+            url: https://github.com/CentOS/container-index
+    triggers:
+        - github-pull-request:
+            admin-list:
+              - kbsingh
+              - bamachrn
+              - mohammedzee1000
+              - dharmit
+              - navidshaikh
+            white-list:
+              - kbsingh
+              - bamachrn
+              - mohammedzee1000
+              - dharmit
+              - navidshaikh
+            cron: '* * * * *'
+            trigger-phrase: '#dotests'
+            only-trigger-phrase: false
+            github-hooks: true
+            permit-all: false
+            auto-close-on-fail: false
+            status-context: "centos-ci container-index ci tests"
+            started-status: "centos-ci container-index ci  tests started"
+            success-status: "centos-ci container-index ci  tests succeeded"
+            failure-status: "centos-ci container-index ci  tests failed"
+            error-status: "centos-ci container-index ci  tests errored"
+    wrappers:
+        - ansicolor
+    scm:
+        - git:
+            url: https://github.com/CentOS/container-index.git
+            skip-tag: True
+            git-tool: ci-git
+            branches: master
+            refspec: "+refs/pull/*:refs/remotes/origin/pr/*"
+            branches:
+                - "${ghprbactualcommit}"
+    builders:
+        - shell: |
+            git rebase origin/${ghprbTargetBranch}
+            curl https://raw.githubusercontent.com/CentOS/container-pipeline-service/master/ci/cccp_ci_container_index.sh > cccp_ci_container_index.sh
```

Comment:
Out of curiosity, how do we handle a situation where the script being downloaded here itself needs to be changed?

Reply:
Yes, in this situation it will cause an issue for testing before merge. Although this script should not need to change very often, it is a possibility.

Reply:
This is container-index CI, and the script itself downloads/clones all the required code base. So, whenever there is a change in the script, it will be tracked with the container-pipeline ci, not this one. @mohammedzee1000 I believe you can give us better explanation, if required on this.